man/systemd.exec.xml

```diff
@@ -519,7 +519,10 @@
         <filename>/opt/</filename> hierarchies for sysext images and <filename>/etc/</filename>
         hierarchy for confext images. The order in which the images are listed will determine the
         order in which the overlay is laid down: images specified first to last will result in overlayfs
-        layers bottom to top.</para>
+        layers bottom to top. If multiple identical dm-verity protected images (i.e. with the same roothash)
+        are specified, the first one is retained and the following ones are pruned, as OverlayFS does not
+        allow referencing same directory inode as determined by filesystem UUID and file handle multiple
+        times in <option>lowerdir=</option>.</para>
```

Comment:
so, i think this is still misleading. Because this only applies to legacy verity stuff, where things are split up. DDIs are not covered by this. This wording suggests we'd always do this.

In my view this is an optimization only. Hence no need to document this at all I'd say.

Reply:
Sure that's fine by me, dropped